index.js

```diff
@@ -1,68 +0,0 @@
-const { strict } = require("assert");
-const readline = require("readline");
-const rl = readline.createInterface(process.stdin, process.stdout);
-
-function ask(questionText) {
-  return new Promise((resolve, reject) => {
-    rl.question(questionText, resolve);
-  });
-}
-
-start();
-
-async function start() {
-  console.log(
-    "Let's play a game where you (human) make up a number and I (computer) try to guess it."
-  );
-  let secretNumber = await ask(
-    "What is your secret number?\nI won't peek, I promise...\n"
-  );
-  console.log("You entered: " + secretNumber);
-
-  //get a random number between 0,100 and store it as 'guessTheNumber'
-  function getRandomInt(min, max) {
-    let range = max - min + 1;
-    return min + Math.floor(Math.random() * range);
-  }
-  guessTheNumber = getRandomInt(1, 100);
-  min = 1;
-  max = 100;
-  let response = await ask("Is your number? " + guessTheNumber);
-  //player responds yes and gets a victory message
-  if (response === "yes" || response === "y") {
-    console.log("You Win!");
-  }
-
-  //computer responds with "is your number higher or lower"
-
-  while (response === "no" || response === "n") {
-    let highLow = await ask("Is your number higher or lower? ");
-    console.log(highLow);
-    if (highLow === "higher" || highLow === "h") {
-      guessTheNumber = getRandomInt(guessTheNumber, max);
-      console.log(highLow);
-      min = guessTheNumber + 1;
-      console.log(max, min, guessTheNumber);
```

Comment:
This is the right direction, but there's some weird things happening through this section. (and with the lower section too). The order that things happen matters a lot through out code, what happens here is that when the user says "higher", the computer generates a new random number, then redefines `min` as the new number +1 and doing this will cause your range to now be between two numbers. You can try changing the order that things happen in here and it should fix a couple of the bugs that are going on.